scripts/boxesserver

```diff
@@ -341,7 +342,7 @@ class BServer:
 """ ]
         for nr, group in enumerate(self.groups):
             result.append(f'''
-<h3 id="h-{nr}" class="open" onclick="showHide('{nr}')"
+<h3 id="h-{nr}" class="open" tabindex="0" onclick="showHide('{nr}')" onkeypress="if(event.keyCode == 13) showHide('{nr}')"
```

Comment:
Adds event handler for keyboard enter press and tab index to allow for tab navigation on clickable headings and a more device independent event handler.